src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/ModelMetadataDisplayDetails.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public class ModelMetadataDisplayDetails
```

Comment:
Mostly. `IsRequired` and `IsReadOnly` are kinda special-casey - we need to compute them based on the type/property. So what I do here is default them to null and then treat it as an 'override'. Open to suggestions that don't significantly complexify the code if you think it's bad.